spec/features/admin/export_advisers_list_spec.rb

```diff
@@ -0,0 +1,96 @@
+# Feature: Advisors list export
+#  As the person maintaining the RAD database.
+#  In order to get rid of unwanted advisors.
+#  I need to be able to cross reference a large list (4k) of unauthorised advisers against records in the RAD database.
+#  The list needs to be cleared up so customers do not get sent to unapproved advisers and RAD maintain it's credibility.
+#
```

Comment:
I'm not sure the rationale for the feature should be included in the code, but as we have no standards, your call. We can discuss in next week's meeting re Developer Processes/Workflow.